### ISSUE TITLE
Imports in diffusion_policy_state_pusht_demo notebook fail in Google Colab runtime

### ISSUE THREAD

**Issue by alex-klimov** [real-stanford/diffusion_policy#26]
Hello guys,

Very impressive work. Would love to try it out with [state-based notebook](https://colab.research.google.com/drive/1gxdkgRVfM55zihY9TFLja97cSVZOZq2B?usp=sharing), but fail to run in Google Colab runtime. Import cell fails with:
```
XlaRuntimeError: FAILED_PRECONDITION: DNN library initialization failed. Look at the errors above for more details.
```
on this line:
```Python
from diffusers.training_utils import EMAModel
```

Could you please advise on fix/workaround?

**Comment by cheng-chi** [comment]
Hi @alex-klimov I have updated the notebooks.
Fixed by updating diffusers to 0.18.2

**Comment by alex-klimov** [comment]
Thank you! I will check it out soon. I got the project to work on my local machine, so not blocked on my side.
